db/models.py

```diff
@@ -1 +1,43 @@
 from django.db import models
+
+
+class Race(models.Model):
+    race_to_choose = [
+        ["Elf", "Elf"],
+        ["Dwarf", "Dwarf"],
+        ["Human", "Human"],
+        ["Ork", "Ork"]
+    ]
```

Comment:
The `choices` attribute should use tuples instead of lists for each choice. Change each list to a tuple, e.g., `('Elf', 'Elf')`.